Scheduler/feature_scheduler/auxtel/feb_2025_fbs_spec_survey.py

```diff
@@ -0,0 +1,205 @@
+# This file is part of ts_config_ocs.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from astropy import units
+from astropy.coordinates import Angle
+from lsst.ts.fbs.utils import Target
+from lsst.ts.fbs.utils.auxtel.surveys import (
+    generate_cwfs_survey,
+    generate_image_survey_from_target,
+    generate_spectroscopic_survey,
+    get_auxtel_targets,
+)
+from rubin_scheduler.scheduler.detailers import DitherDetailer
+from rubin_scheduler.scheduler.schedulers import CoreScheduler
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+    # Information for all surveys
+    nside = 64
```

Comment:
Since we're just using FieldSurveys and don't have any basis functions that require spatial resolution on the scale of the detector FOV, I wonder if we can just go back to the default nside=32 here...

Reply:
I will check. I was thinking, if 64 wasn't being too slow, you could stick with that. However, 32 is even faster and makes pickles much smaller. 
So it'd be good to go down .. the only thing I was thinking is that pixel size of the maps like the avoidance zones is a little larger, so you'll end up masking a bit more than otherwise. But I suppose that's not really that much different, so yeah-  I'll check with nside 32.